resources/frameworks.yaml

```
@@ -69,6 +69,21 @@ AutoWEKA:
   version: '2.6'
   project: https://www.cs.ubc.ca/labs/beta/Projects/autoweka/
 
+MLPlan:
+  module: frameworks.MLPlan
+  version: '0.2.3'
+  project: https://mlplan.org
```

just noticed that this URL currently points to the Reisebuch from the author :)
Apparently, it's not the case with the `http` version which redirects correctly.
```suggestion
  project: http://mlplan.org
```